docs/rules/prefer-focus.md

```diff
@@ -35,9 +35,9 @@ expect(foo).not.toEqual(document.activeElement);
 Examples of **correct** code for this rule:
 
 ```js
-expect(document.activeElement).not.toBeNull();
+expect(document.activeElement).not.toHaveFocus();
 
-expect(document.activeElement).toBeNull();
+expect(document.activeElement).toHaveFocus();
```

Comment:
I'm a bit confused, these are actually valid as you can see in the tests: 

https://github.com/testing-library/eslint-plugin-jest-dom/blob/master/src/__tests__/lib/rules/prefer-focus.js#L12-L13

`expect(document.activeElement).not.toBeNull()` is just a check that 'something is focused and I don't care what'
`expect(document.activeElement).toBeNull()` is a check that 'nothing is focused'.

did you run into an issue or something? just curious.

Reply:
I thought the correct examples were wrong. I was expecting to see correct examples by using `toHaveFocus` which is what the rule description suggest. I guess I can add the new examples rather than replace current.

Reply:
ah ok fair enough.  can you add them to the tests as well please? :)

Reply:
actually after looking at it, i wonder if we should actually report on `expect(document.activeElement).toHaveFocus();` as it's redundant.  `toBeFocused()` is checking `element.ownerDocument.activeElement === element` so having this expect is essentially asserting that document.active elemnent === itself.